src/react-popper.js

```diff
@@ -1,4 +1,4 @@
-export { default as Manager } from './Manager'
-export { default as Target } from './Target'
-export { default as Popper } from './Popper'
-export { default as Arrow } from './Arrow'
+export Manager from './Manager'
```

Comment:
Is this valid syntax?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export
https://stackoverflow.com/q/41139740/1873485

Reply:
Only with babel for the moment: https://babeljs.io/docs/plugins/transform-export-extensions/